### ISSUE TITLE
.ahk file backup and upload process not working

### ISSUE THREAD

**Issue by mumos0** [elModo7/Wiimote-Remapper-AHK#1]
I have tried several ways to be able to edit each of the commands that each button on my wiimote can execute and came to the conclusion that it is failing to load assigned buttons.

The only reliable way so far would be to execute the hacker resource to the .exe file and the RCDATA folder and change the actions that each button does.

example

Up:
SendInput {up}
return
Sub:
Send, {Volume_Down}
return
Add:
Send, {Volume_Up}
return
1:
SendInput {v}
return
2:
Run, https://twitter.com/
return
A:
MsgBox, New Message.
return

You can change all that according to the ahk configurations

Lastly, I don't have much programming knowledge to be able to fix the loading issue so that the .ank files can run reliably.

**Comment by elModo7** [comment]
True, I am thinking on reworking this project when I have some spare time.
